src/Helpers/Coverage.php

```diff
@@ -8,15 +8,15 @@
 
 class Coverage
 {
-    public static function fromXml(SimpleXMLElement $xml): float
+    public static function fromXml(SimpleXMLElement $xml, string $metricToParse = 'elements'): float
```

Comment:
I think it's not necessary to add the default value here, this should come from the parsed arguments where elements is the default value

Reply:
Right. but will need to change the rest of tests to send this default argument. What do you prefer?

Reply:
Done